lib/sentry/plug_context.ex

```diff
@@ -70,11 +70,11 @@ defmodule Sentry.PlugContext do
 
   Then, pass it into `Sentry.PlugContext`:
 
-      plug Sentry.PlugContext, header_scrubber: &MyHeaderScrubber.scrub_headers/1
+      plug Sentry.PlugContext, header_scrubber: &MySentryScrubber.scrub_headers/1
 
   It can also be passed in as a `{module, fun}` like so:
 
-      plug Sentry.PlugContext, header_scrubber: {MyModule, :scrub_headers}
+      plug Sentry.PlugContext, header_scrubber: {MySentryScrubber, :scrub_headers}
```

Comment:
It's an insignificant detail, but there was some inconsistency with what module names were used. Since I was updating related docs and wanted to be consistent, thought it felt clean to set them all to `MySentryScrubber`.

Happy to revert if this isn't preferred!